layers/+source-control/version-control/README.org

```diff
@@ -211,12 +211,12 @@ Use ~SPC g .~ to enter a transient state for quickly navigating between hunks in
 ** Smerge Mode Transient-state
 Movement:
 
-| Key binding                | Description         |
-|----------------------------+---------------------|
-| ~SPC g r n~                | Next hunk           |
-| ~SPC g r N~ or ~SPC g r p~ | Previous hunk       |
-| ~SPC g r j~                | Go to next line     |
-| ~SPC g r k~                | Go to previous line |
+| Key binding                | Description                              |
+|----------------------------+------------------------------------------|
+| ~SPC g r n~                | Next conflict (possibly in another file) |
```

Comment:
I think replacing `hunk` by `conflict` is fine at least, if it's not clearer.

But why do you added `(possibly in another file)`? And not adding that to `SPC g r N` or `SPC g r p`?

Reply:
I wanted to highlight the fact that this command now may take you to another file. `n`/`P` is unchanged and only looks within the file.

Reply:
Maybe I should add that to the transient mode hint as well...

Reply:
I'm not sure about this.
Usually I prefer clarity, but does it really cause confusion if you didn't add `(possibly in another file)`?

Reply:
Well, people might not discover that they can go between files otherwise. They'd just have to happen to hit `n` after the last conflict.